apps/integration/integration/automation/runtests.sh

```diff
@@ -39,5 +39,14 @@ handle_outputs() {
   echo "Yep, definitely handling test outputs. That's what this function does" >&2
 }
 
+runtests() {
+  docker system prune -f --volumes
+  docker compose up reset
+  poetry run pytest apps/integration/ -p integration.conftest --noconftest --docker-tag latest_rc
+  # this is a complicated command, so: ^                        ^            ^ test against containers tagged latest_rc
+  #                                    |                     don't load conftest at pytest runtime; it's already loaded
+  #                                     load conftest with plugins, to capture the custom command line arg --docker-tag
```

Comment:
Nice hack.  Not sure the ASCII art translates to narrower screens.

Reply:
I'll let you take up the 80 versus 120 argument with someone else.